bin/coreos-install

```diff
@@ -332,6 +341,29 @@ if [[ -n "${CLOUDINIT}" ]] || [[ -n "${COPY_NET}" ]]; then
     fi
 
     umount "${WORKDIR}/rootfs"
+    trap "rm -rf '${WORKDIR}'" EXIT
+fi
+
+if [[ -n "${IGNITION}" ]]; then
+    # The OEM partition should be #3 but make no assumptions here!
+    # Also don't mount by label directly in case other devices conflict.
+    OEM_DEV=$(blkid -t "LABEL=OEM" -o device "${DEVICE}"*)
+
+    if [[ -z "${OEM_DEV}" ]]; then
+      echo "Unable to find new OEM partition on ${DEVICE}" >&2
+      exit 1
+    fi
+
+    mkdir -p "${WORKDIR}/oemfs"
+    mount "${OEM_DEV}" "${WORKDIR}/oemfs"
+    trap "umount '${WORKDIR}/oemfs' && rm -rf '${WORKDIR}'" EXIT
+
+    echo "Installing Ignition config ${IGNITION}..."
+    cp "${IGNITION}" "${WORKDIR}/oemfs/ignition.json"
```

Comment:
Let's go ahead and name this "coreos-install.json". That way, we'll know where the config came from in the future.